src/modules/render.js

```diff
@@ -0,0 +1,24 @@
+// -- function that renders the information coming from the API.
+const cards = document.querySelector('.cards');
+const displayPhotos = async (photos) => {
+  for (let i = 0; i < photos.length; i += 1) {
```

Comment:
This function is not async function, Hence you can remove async keyword.
But, If you follow my comment in api.js file, you want to involve getData function inside this code, you can use it. That would be ok.